public/app/plugins/panel/canvas/migrations.ts

```diff
@@ -42,5 +43,44 @@ export const canvasMigrationHandler = (panel: PanelModel): Partial<Options> => {
     }
   }
 
+  for (let idx = 0; idx < panel.fieldConfig.overrides.length; idx++) {
```

Comment:
I think we need to wrap this with a conditional based on pluginVersion so it isn't ran everytime but only for when this change is included 🤔 

something like
```
if (pluginVersion.startsWith('11.2')) {
  ...
}
```

Testing migration locally then can be accomplished via manually modifying the dashboard json version to < current version - happy to to pair on this :)

Reply:
I'm not sure, I think this migration goes beyond 11.2, as the users can still add data links overrides that we'll need to migrate to elements.

Reply:
That case wouldn't be covered by this migration code as the migration code only triggers when panel version != version i'm pretty sure 😅